python/desc/sims/GCRCatSimInterface/generate_lensed_hosts_agn.py

```diff
@@ -0,0 +1,232 @@
+#matplotlib inline
+#config InlineBackend.figure_format = 'retina'
+
+import numpy as np
+import os
+import pylab as pl
+import subprocess as sp
+import astropy.io.fits as pyfits
+import pandas as pd
+import scipy.special as ss
+import om10_lensing_equations as ole
+
+outdir = "./outputs/"
+data_dir = os.path.join(os.path.abspath("../../../../"),'data')
+twinkles_data_dir = os.path.join(os.environ['TWINKLES_DIR'], 'data')
+
+def load_in_data_agn():
+
```

Comment:
Please add docstrings to all of the methods in these scripts.  It will make them easier to maintain/debug in the future.